lib/amqp-ts.d.ts

```diff
@@ -108,7 +108,7 @@ export declare class Exchange {
      */
     publish(content: any, routingKey?: string, options?: any): void;
     send(message: Message, routingKey?: string): void;
-    rpc(requestParameters: any, routingKey?: string, callback?: (err, message: Message) => void): Promise<Message>;
+    rpc(requestParameters: any, routingKey?: string, callback?: (err: any, message: Message) => void): Promise<Message>;
```

Comment:
should this be `Error`?

Reply:
I did not check what the correct type would be. But having the correct types an not any would be preferable.